suite-common/message-system/src/messageSystemTypes.ts

```diff
@@ -83,6 +83,9 @@ const getTradingContext = (type: TradingType) => `trading.${type}` as const;
 export type SettingsCategory = 'general' | 'device' | 'networks' | 'debug';
 const getSettingsContext = (category: SettingsCategory) => `settings.${category}` as const;
 
+export type LegalContextKey = 'gateway';
+const getLegalContext = (key: LegalContextKey) => `legal.${key}` as const;
```

Comment:
```suggestion
const getLegalContext = (key: LegalContextKey) => `legal.${key}` as const;
```

How about `staking.footer.legal` and `trading.footer.legal` or something like that. I dont like calling it `legal` 😀